apps/api/src/controllers/v1/search.ts

```
@@ -0,0 +1,260 @@
+      description: searchResult.description,
+      url: searchResult.url,
+      metadata: {
+        statusCode,
+        error: error.message,
+      },
+    };
+  }
+}
+
+export async function searchController(
+  req: RequestWithAuth<{}, SearchResponse, SearchRequest>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)